--- gnustep-guile-1.0.3/Documentation/GNUmakefile.orig	Tue Feb  5 04:50:41 2002
+++ gnustep-guile-1.0.3/Documentation/GNUmakefile	Tue Dec 31 14:17:34 2002
@@ -12,18 +12,23 @@
 
 # The texinfo source files to be used
 gstep_guile_TEXI_FILES = install.texi news.texi gstep_guile.texi
+gstep_guile_DOC_INSTALL_DIR = Developer/Guile/Manual
 
 README_TEXI_FILES = version.texi
 README_TEXT_MAIN = readme.texi
+README_DOC_INSTALL_DIR = Developer/Guile/ReleaseNotes/$(VERSION)
 
 INSTALL_TEXT_MAIN = install.texi
 INSTALL_TEXI_FILES = version.texi
+INSTALL_DOC_INSTALL_DIR = Developer/Guile/ReleaseNotes/$(VERSION)
 
 NEWS_TEXT_MAIN = news.texi
 NEWS_TEXI_FILES = version.texi
+NEWS_DOC_INSTALL_DIR = Developer/Guile/ReleaseNotes/$(VERSION)
 
 ANNOUNCE_TEXT_MAIN = announce.texi
 ANNOUNCE_TEXI_FILES = version.texi
+ANNOUNCE_DOC_INSTALL_DIR = Developer/Guile/ReleaseNotes/$(VERSION)
 
 -include Makefile.preamble
 
